hoomd/hpmc/pytest/test_compute_free_volume.py

```diff
@@ -76,6 +76,14 @@ def test_validation_systems(simulation_factory, two_particle_snapshot_factory,
                                free_volume_compute.free_volume,
                                rtol=2e-2)
 
+    # Tet the kernel parameter tuner.
+    def activate_tuner():
+        sim.run(1)
+        return free_volume_compute.free_volume
```

Comment:
No need for the return statement
```suggestion
        # We need to make the kernel be called.
        free_volume_compute.free_volume
```

Reply:
I'll try this change. I didn't write it this way as I expected flake8 to complain.